cpu/esp32/Kconfig.esp32s2

```diff
@@ -26,7 +26,7 @@ config CPU_FAM_ESP32S2
     select MODULE_PS if MODULE_SHELL
     select MODULE_PTHREAD if MODULE_CPP
     select MODULE_RTT_RTC if HAS_PERIPH_RTT && MODULE_PERIPH_RTC
-    select MODULE_USBDEV_SYNOPSYS_DWC2 if HAS_PERIPH_USBDEV && MODULE_PERIPH_USBDEV
```

Comment:
It would be better to split ESP32 to another PR.
Changes are rather small so this should be quick.